### DISCUSSION TITLE
404 with a succesful build

### DISCUSSION THREAD

**Discussion by ridheshcybe** [vercel/community#7368] [Account Issue]
### Summary

i got a 404 with a succesful build with this vercel.json [vercel.json](https://github.com/user-attachments/files/16101763/vercel.json)
![image](https://github.com/vercel/community/assets/80441770/5e242729-ffb7-4169-a0db-d2ab5083d3e8)

### Example

https://track50.vercel.app/

### Steps to Reproduce

go to https://track50.vercel.app/

**Comment by Padaipaaligal** [comment]
Hi @ridheshcybe, in my case, when I reload the page except '/', I'm getting the same error.
Steps to reproduce.
1. Go to https://fabric-design.vercel.app/
2. Click login cta on the top right side
3. Reload the page
Or directly go to the page https://fabric-design.vercel.app/login.

<img width="463" alt="image" src="https://github.com/vercel/community/assets/172921081/00b5b163-e73c-4e52-8244-66390fecd717">

<img width="1670" alt="image" src="https://github.com/vercel/community/assets/172921081/889e4649-5918-4d64-ac7f-f34812d68a5d">

Pls help me to resolve this issue. Thanks in Advance!
Note: I have developed this project using vue3.

**Reply by Padaipaaligal** [reply]
@versecafe Do you have any idea on this?

**Comment by versecafe** [comment]
@ridheshcybe If you're deploying a framework that is not officially supported you need to make sure to correctly configure a Vercel JSON for routing behaviour, example for ExpressJS
```json
{ "version": 2, "rewrites": [{ "source": "/(.*)", "destination": "/api" }] }
```

**Comment by ridheshcybe** [comment]
I fixed it it was due to "/index.js" was missing attached is the full code
[vercel.json](https://github.com/user-attachments/files/16111386/vercel.json)

[a comment by an automated account is not shown]
